pyproject.toml

```diff
@@ -4,7 +4,7 @@ requires = [
     "versioneer[toml]",
     "wheel",
     "numpy~=1.25",
-    "Cython~=0.29.36",
+    "Cython>0.29",
```

Comment:
I think this should be 
```suggestion
    "Cython=3",
```
?

Reply:
Done.

Reply:
Unfortunately, this didn't work. 

I tried multiple things, only `Cython>=3.0.0` worked.

Reply:
Weird, I find TOML formatting to be a headache. If it works it works 🤷